stdlib/LibGit2/test/libgit2.jl

```diff
@@ -125,7 +125,7 @@ end
 function get_global_dir()
     buf = Ref(LibGit2.Buffer())
     LibGit2.@check ccall((:git_libgit2_opts, :libgit2), Cint,
-                         (Cint, Cint, Ptr{LibGit2.Buffer}),
```

Comment:
This signature is `git_libgit2_opts(int option, ...);`, which doesn't quite match this. It may not make a difference here, but it would be better to be consistent.

Reply:
Yeah..., the general `...` is hard to express using `ccall`. Especially since we want to have the types to do argument conversion. I suspect that the correct signature is `(Cint, Clong...)` and then manually do the conversion to `Ptr{LibGit2.Buffer}` + reinterpret to `Clong`.  

Any better ideas? For PPC it's only important that the callsite is marked `va`.